packages/investor/config-overrides.js

```diff
@@ -1,4 +1,14 @@
 const rewireYarnWorkspaces = require("react-app-rewire-yarn-workspaces");
```

Comment:
```suggestion
const rewireYarnWorkspaces = require("react-app-rewire-yarn-workspaces");
module.exports = function override(config, env) {
  return rewireYarnWorkspaces(config, env);
};
```
I think that a simple config will be enough